omod/src/main/java/org/openmrs/module/reporting/web/datasets/LogicDataSetEditor.java

```diff
@@ -37,10 +41,13 @@
 	Log log = LogFactory.getLog(getClass());
 	
 	@RequestMapping("/module/reporting/datasets/logicDataSetEditor")
-	public void showDataset(ModelMap model, @RequestParam(required = false, value = "uuid") String uuid) {
+	public void showDataset(HttpServletRequest request,ModelMap model, @RequestParam(required = false, value = "uuid") String uuid) {
```

Comment:
@mozzy11 , kindly format your code changes following the code styles 
at https://wiki.openmrs.org/display/docs/Java+Conventions .

Reply:
@Ruhanga , i still haven't figured out the exact problem , can you please be more specific??

Reply:
It's evident to me that there is no spacing between the first and second parameter definitions after the comma.

Reply:
haha, ok thank you @Ruhanga . but you should have made it clear the first time , not to make me read the whole java convetions document ..anyway thanks . let me work on it